Lib/asyncio/base_events.py

```diff
@@ -303,6 +303,8 @@ def _detach(self, transport):
             self._wakeup()
 
     def _wakeup(self):
+        if self._waiters is None:
```

Comment:
Might as well do the None check on the `waiters` local

Reply:
Addressed in d96323752a277d48d6a173576ed62ce36bd0517b